utils/browser.py

```diff
@@ -62,6 +62,10 @@ def ensure_browser_open():
 
     """
     try:
+        if browser_name:
```

Comment:
As this won't have access to the browser_name before as above, we can always read the conf here, just like we do in ```start()```. I guess we can't delegate the quitting to ```start```....that actually is cleaner as start() is already called in this function, but I guess we need to intercept as browser().current_url would still return something even if it was the wrong browser.